item-spec/item-spec.md

```diff
@@ -111,12 +111,12 @@ whenever possible. But Static Catalogs are potentially more portable if they can
 relative links, so that every link doesn't need to be rewritten when the data is copied. The `self`
 link is required to be absolute.
 
-#### Datasets
+#### Collections
 
-Items are *strongly recommended* to provide a link to a dataset definition. It is important as datasets 
-provide additional information about a set of items, for example the license and provider information 
-and optionally any common information shared across all items, giving context on the overall set of
-data that an individual Item is a part of..
+Items are *strongly recommended* to provide a link to a STAC Collection definition. It is important as Collections 
```

Comment:
Items are required now to link to a Collection

Reply:
That's not completely true, see my comment above ( https://github.com/radiantearth/stac-spec/pull/280#discussion_r224720185 ) - but I improved the wording here.